package.json

```diff
@@ -20,7 +20,8 @@
         "docs": "typedoc --options typedoc.json",
         "docs:clean": "rm -rf docs && mkdir docs && touch docs/.nojekyll && yarn docs",
         "release": "yarn clean && yarn lint && yarn build && yarn publish && yarn release:changelog",
-        "release:changelog": "sh script/changelog.sh"
+        "release:changelog": "sh script/changelog.sh",
+        "prepare": "yarn build"
```

Comment:
Small nit, but can you make this alphabetical?

Reply:
sure - fixed in [replacement PR](https://github.com/actions-on-google/actions-on-google-testing-nodejs/pull/50) (used wrong email in commit)